WORKSPACE

```diff
@@ -425,6 +425,14 @@ maven_install(
     ],
 )
 
+maven_install(
+    name = "service_indexing_testing",
```

Comment:
I am mildly surprised this doesn't need to be included in the `use_repo` for `dev_maven` further down this file (around line 665ish). Because we now alias `pin` into the generated workspace, you won't need to include the `unpinned_` version there too.

Reply:
I believe this is included in the `use_repo` for `dev_maven` (currently on line 687) - is that not what you mean?